src/IssueViz/IssueVizPackage.cs

```diff
@@ -37,12 +39,17 @@ namespace SonarLint.VisualStudio.IssueVisualization
     [ProvideToolWindowVisibility(typeof(IssueVisualizationToolWindow), Constants.UIContextGuid)]
     public sealed class IssueVizPackage : AsyncPackage
     {
+        private IFileRenamesEventHandler fileRenamesEventHandler;
```

Comment:
We won't start tracking file renames until the package is loaded, which won't happen until either a command is called or the tool window is opened.

We either need to change when the package loads, or instantiate the rename tracker in a different way. We need the tracker from the point that a file is analysed/issues pushed to the error list, so we could create it in part of the analysis infrastructure.